frontend/src/components/common/table/FilterRow.tsx

```diff
@@ -0,0 +1,61 @@
+import React from "react";
+import { Flex, Text } from "@chakra-ui/react";
+import { Select } from "chakra-react-select";
+
+import type { FilterProp } from "./FilterMenu";
+
+interface FilterRowProps {
+  filterProp: FilterProp;
+  optionsSelected: Map<string, string[]>;
+  setOptionsSelected: React.Dispatch<
+    React.SetStateAction<Map<string, string[]>>
+  >;
+}
+
+const FilterRow = ({
+  filterProp,
+  optionsSelected,
+  setOptionsSelected,
+}: FilterRowProps): React.ReactElement => {
+  return (
+    <>
+      <Flex justifyContent="space-around" pb={4} pt={2} px={3}>
+        <div style={{ width: "7rem" }}>
+          <Text color="blue.300" textStyle="link">
+            {filterProp.label}
+          </Text>
+        </div>
+
+        <div style={{ width: "23rem" }}>
+          <Select
+            chakraStyles={{
+              dropdownIndicator: (provided) => ({
+                ...provided,
+                background: "white",
+              }),
+              downChevron: (provided) => ({
+                ...provided,
+                boxSize: "1.5rem",
+              }),
+              crossIcon: (provided) => ({
+                ...provided,
+                boxSize: "0.8rem",
+              }),
+            }}
+            focusBorderColor="blue.300"
+            isMulti
+            onChange={(choices) => {
```

Comment:
@joyce-shi On the last PR we discussed moving styling for` <Select>` into the `themes` folder.

So Select is not a native Chakra component, but rather a wrapper component imported from chakra-react-select, and so the default theme doesn't work on it from what I'm looking into. In the [chakra-react-select docs](https://www.npmjs.com/package/chakra-react-select#theme-styles), it describes how individual components can be altered, but it says to use chakrastyles if you don't want to affect all components.

"Only make changes to your global component themes if you want them to appear in all instances of that component. Otherwise, just change the individual components' styles using the chakraStyles prop."

I'm not how to approach the styling, but I'd love to hear thoughts!